nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        Y_hat_df = forecast.reset_index()
</pre><p>Please add a cell at the top with the following:</p>

<pre class="ql-syntax" spellcheck="false">import os
os.environ['NIXTLA_ID_AS_COL'] = '1'
</pre><p>that way you won't need to reset the index here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Added cell at the top as suggested and removed the .reset_index() method.</p>